components/ColorPickerModal.js

```diff
@@ -44,14 +44,7 @@ export default function ColorPickerModal({ initialColor, onColorSelected, onClos
     {
       key: 'basic',
       title: 'Standard',
-      component: (
-        <CromaColorPicker
-          onChangeColor={(color) => {
-            setColor(color);
-          }}
-          style={[{ height: 250 }]}
-        />
-      )
+      component: <WheelColorPicker color={color} onColorChange={(color) => setColor(color)} />
```

Comment:
Hi @diegosoriarios ,
Thanks for the contribution. This should be added as a new tab in the color picker. We can probabaly name it `color wheel` 
Also can you please add screenshots or a small video of testing this feature?

Reply:
Hello, I fix that and create a new tab.
And here a video of me testing the featuer.

https://github.com/croma-app/huehive-mobile-app/assets/24478606/c0a9f2fa-7d64-4dce-9c55-1d6abe802dd2